### PR TITLE
Pull MATE 1.18.x updates from kenmays/oi-userland

### PR DESCRIPTION
**Pull request by kenmays**
Pull 2017-05-08

### CODE REVIEW

**Comment by alarcher** [comment]
Caja 1.16.5 requires mate-desktop-2.0 >= 1.16.2 so the build will fail on the server if the PR is merged.
It should be put in a subsequent PR.

**Comment by kenmays** [comment]
Mate-desktop is defined for version 1.16.2.https://github.com/kenmays/oi-userland/blob/oi/hipster/components/desktop/mate/mate-desktop/Makefile


On Monday, May 8, 2017, 5:53:44 PM PDT, Aurélien Larcher <notifications@github.com> wrote:
Caja 1.16.5 requires mate-desktop-2.0 >= 1.16.2 so the build will fail on the server if the PR is merged.
It should be put in a subsequent PR.

**Comment by alarcher** [comment]
Exactly, as soon as the PR is merged caja will be built in an environment where mate-desktop 1.16.1 only is installed so it will fail:
- a first PR should include mate-desktop 1.16.2 so that it is built then installed on the build server,
- then a second PR will update caja which can now be built in the environment containing newly installed mate-desktop.
Keep it mind that the components are only installed after all of them have been published.
Thank you :)

**Review by alarcher — CHANGES_REQUESTED** [review]
Build done and tested.
Merge commit should be removed and Caja should pe put in a second PR.

**Comment by kenmays** [comment]
Closing this pull request for merge corrections.

**Comment by alarcher** [comment]
@kenmays Keep it open, it is fine.

**Comment by alarcher** [comment]
@kenmays Do like this:
git checkout oi/hipster
git checkout -b caja
to keep your caja commit in a branch.

git checkout oi/hipster
git rebase -i HEAD~9

then remove the lines with the merge commit and caja

git push -f 

to push again you branch, the PR will be updated automatically.

File: components/desktop/mate/mate-terminal/Makefile [hunk 1]
```diff
@@ -10,19 +10,20 @@
...
 # Copyright 2016 Alexander Pyhalov
+# Copyright 2017 Ken Mays
 #
 
 include ../../../../make-rules/shared-macros.mk
 
 COMPONENT_NAME=		mate-terminal
-COMPONENT_VERSION=      1.16.1
+COMPONENT_VERSION=      1.16.3
 COMPONENT_REVISION=	1
```
Comment (pyhalov) [inline components/desktop/mate/mate-terminal/Makefile]: Please, remove COMPONENT_REVISION while updating COMPONENT_VERSION

File: components/desktop/mate/mate-control-center/Makefile [hunk 2]
```diff
@@ -15,7 +15,7 @@
 include ../../../../make-rules/shared-macros.mk
 
 COMPONENT_NAME=		mate-control-center
-COMPONENT_VERSION=	1.16.0
+COMPONENT_VERSION=	1.16.1
```
Comment (pyhalov) [inline components/desktop/mate/mate-control-center/Makefile]: COMPONENT_ARCHIVE_HASH is not updated

File: components/desktop/mate/mate-themes/Makefile [hunk 3]
```diff
@@ -16,13 +16,13 @@
...
-COMPONENT_VERSION=	3.20.7
+COMPONENT_VERSION=	3.20.19
 COMPONENT_PROJECT_URL=	http://www.mate-desktop.org
 COMPONENT_SUMMARY=	MATE Themes
 COMPONENT_SRC=		$(COMPONENT_NAME)-$(COMPONENT_VERSION)
 COMPONENT_ARCHIVE=	$(COMPONENT_SRC).tar.xz
 COMPONENT_ARCHIVE_HASH=	\
-    sha256:c2909366b83e9e56e630fd5f51c5cf2f6993e0eb337717cac2663c708577606e
+    sha256:0fad212926a59aedbeaf45d5e96a2337cebd54cc17ced38f5e50ce46ce786885 
 COMPONENT_ARCHIVE_URL=	http://pub.mate-desktop.org/releases/themes/3.20/$(COMPONENT_ARCHIVE)
```
Comment (pyhalov) [inline components/desktop/mate/mate-themes/Makefile]: Doesn't publish:
`` Couldn't find 'usr/share/themes/Menta/gtk-3.0/assets/slider-hover.png' in any of the specified search directories:``
and so on. Have you updated manifests and sample-manifests?

**Comment by kenmays** [comment]
Latest updates are at MATE 1.18 (2017-05-15). The older ones are updated for the latest MATE 1.16 updates to fix previous pull request by Alex.

**Comment by alarcher** [comment]
@kenmays A few missing build dependencies were fixed in this branch https://github.com/OpenIndiana/oi-userland/pull/3014 It would be good to add them.

**Comment by alarcher** [comment]
@kenmays @pyhalov @xen0l @Toasterson Do we want 64bit builds only?

**Comment by kenmays** [comment]
MATE 1.18.x 64-bit eventually..

**Comment by pyhalov** [comment]
For binaries - likely

**Comment by Toasterson** [comment]
I would also say for binaries yes and for libs keep 32bit for now.

**Comment by kenmays** [comment]
I converted MATE 1.18 entirely over to 64-bit, You can review from this point.

**Comment by alarcher** [comment]
@kenmays @pyhalov Thank you for your work on MATE! It looks all good!

**Comment by pyhalov** [comment]
Superseded by #3177 , #3178, #3179 , #3181, #3182

**Pull request closed without merging**